rllib/core/learner/learner_group.py

```diff
@@ -128,11 +129,14 @@ def __init__(
 
             # Run the neural network building code on remote workers.
             ray.get([w.build.remote() for w in self._workers])
-            # Use only 1 max in flight request per worker since training workers have to
-            # be synchronously executed.
+
             self._worker_manager = FaultTolerantActorManager(
                 self._workers,
-                max_remote_requests_in_flight_per_actor=1,
+                # TODO (sven): This probably works even without any restriction
+                #  (allowing for any arbitrary number of requests in-flight). Test with
+                #  3 first, then with unlimited, and if both show the same behavior on
+                #  an async algo, remove this restriction entirely.
```

Comment:
I might change this here to "completely unrestricted". Pending benchmarks. I don't think it matters at all, but using 1 or 2 slows down things and deteriorates performance.